020-Getting-started/070-nextjs.mdx

```diff
@@ -50,185 +46,621 @@ apiKey=YOUR_API_KEY_HERE
 
 ## Create a new Next.js app
 
-To start lets create a Next.js application. Set it up with typescript support so that we can utilize the SDK later in this tutorial.
+Let's begin by creating a new Next.js application:
+
+```sh
+npx create-next-app@latest --typescript xata-nextjs
+```
+
+<details>
+  <summary><code>npx create-next-app</code> output</summary>
+
+```sh
+$ npx create-next-app@latest --typescript xata-nextjs
+Need to install the following packages:
+create-next-app@13.4.13
+Ok to proceed? (y)
+✔ Would you like to use ESLint? … No / Yes
+✔ Would you like to use Tailwind CSS? … No / Yes
+✔ Would you like to use `src/` directory? … No / Yes
+✔ Would you like to use App Router? (recommended) … No / Yes
+✔ Would you like to customize the default import alias? … No / Yes
+Creating a new Next.js app in ./xata-nextjs.
+
+Using npm.
+
+Initializing project with template: app-tw
+
+Installing dependencies:
+
+- react
+- react-dom
+- next
+- typescript
+- @types/react
+- @types/node
+- @types/react-dom
+- tailwindcss
+- postcss
+- autoprefixer
+- eslint
+- eslint-config-next
+
+added 314 packages, and audited 315 packages in 22s
+
+107 packages are looking for funding run `npm fund` for details
+
+found 0 vulnerabilities Initialized a git repository.
+
+Success! Created xata-nextjs at ./xata-nextjs
+```
+
+</details>
+
+Once the command has completed, move to the `xata-nextjs` directory and run the application:
 
-```bash
-npx create-next-app@latest --typescript my-xata-app
+```sh
+cd xata-nextjs
+npm run dev
 ```
 
-For the purpose of this tutorial, we're going to assume you named this project `my-xata-app`. Move to that directory, and run `npm run dev` to see your website at http://localhost:3000
+By default, the application will run on `http://localhost:3000`.
 
-### Initialize Xata in your my-xata-app directory
+## Create a new database
 
-Xata needs a few things to communicate with the databases we set up. It needs to know the location of the database, along with the API key it can use to request data from that location. These are stored in the project root directory within `.env` and `.xatarc` respectively. We can use the newly installed CLI to initialize any project with these details.
+With the Xata CLI installed and logged in and a new Next.js application in place, use the Xata CLI to create a new
+database. Accept all the prompt defaults for the following command:
```

Comment:
Nit: "all the prompt defaults" except maybe the region, which should be the closest to the user. Not sure if it's worth pointing that out.